src/vstest.console/app.config

```diff
@@ -21,4 +26,38 @@
     <!-- This flag is added to support test execution for net35 tests through TMI adapter. -->
     <add key="TestProjectRetargetTo35Allowed" value="true" />
   </appSettings>
+
+  <!--
+        To collect logs for vstest.console and it's child processes, follow below steps.
+          1. Uncomment below system.diagnostics element content.
+          2. Replace "c:\tmp\log.txt" with valid filename. Make sure "c:\tmp" folder exists. log.txt file will be created by vstest.console if does not exists.
+          3. Restart vstest.console or it's client(VS, any other IDE).
+        More details: https://github.com/Microsoft/vstest-docs/blob/master/docs/diagnose.md#collect-trace-using-config-file
+  -->
+
+  <!--
+  <system.diagnostics>
+    <sources>
+      <source name="TpTrace"
```

Comment:
nit: We can make this these tags inline, make it look less complicated.

Reply:
Done.